.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports =  {
+    semi:  false,
+    trailingComma:  'all',
+    singleQuote:  true,
+    printWidth:  120,
+    tabWidth:  4,
```

Comment:
Can we keep same tab width as [dex-react](https://github.com/gnosis/dex-react/blob/develop/.prettierrc.js#L6) for consistency across projects?

Reply:
Will make this change after all the PRs are merged. Otherwise prettier will change all files. There will be conflicts.
The waterfall, the horror!